ScienceJournal/UI/CameraViewController.swift

```diff
@@ -374,7 +374,7 @@ open class CameraViewController: ScienceJournalViewController, DrawerItemViewCon
   // MARK: - Private
 
   private func handleImageDataSelected(_ imageData: Data, metadata: NSDictionary?) {
-    delegate?.imageSelectorDidCreateImageData(imageData, metadata: metadata)
+    delegate?.imageSelectorDidCreateImageData([(imageData, metadata)])
```

Comment:
These changes would probably be cleaner/easier to read if you `typealias` the tuple and give it a name, that way it can be used throughout this instead of the () and [()] etc.

Reply:
Oh, great idea 👍

Reply:
I think you missed a few references to the tuple typealias after making that change. Here and one below.